label_persons/data

```diff
@@ -0,0 +1,3 @@
+Image Name,Label
```

Comment:
This file probably should be removed?

Reply:
I didn't mean the header row, that row in label_persons/shane.csv should be kept, I meant this file (label_persons/data).